test/mod/test_buildroot.py

```diff
@@ -28,25 +31,27 @@ def test_basic(self):
         libdir = os.path.abspath(os.curdir)
         var = pathlib.Path(self.tmp.name, "var")
         var.mkdir()
+        timeout = None
```

Comment:
This, and elsewhere could be left out as `timeout=None` is the default, is it not?